gphoto2cffi/backend.py

```diff
@@ -131,6 +123,22 @@ def __getattr__(self, name):
         else:
             return val
 
+    def _logging_callback(self, level, domain, message, data):
```

Comment:
Instance methods didn't worked as I hoped that they would.  I'll make the modifications for #3  and update the pull request.